### PR TITLE
osdocs-441 adding registry storage info to install

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://jira.coreos.com/browse/OSDOCS-441

Built draft here: http://file.rdu.redhat.com/kalexand/060519/osdocs441/installing/installing_vsphere/installing-vsphere.html#installation-overview-installing-vsphere

@huffmanca, @bmcelvee, will you PTAL?

### CODE REVIEW

**Comment by bmcelvee** [comment]
LGTM. I'll hold off on switching the label until Christian has a chance to take a look.

File: modules/installation-registry-storage-non-production.adoc [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // Module included in the following assemblies:
 //
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
 // * installing/installing_bare_metal/installing-bare-metal.adoc
 // * installing/installing_vsphere/installing-vsphere.adoc
 
 [id="installation-registry-storage-non-production_{context}"]
-= Provide storage to the image registry
+= Providing non-production storage to the image registry
```
Comment (huffmanca) [inline modules/installation-registry-storage-non-production.adoc]: This title and section seem to be about configuring an image registry to emptydir for non-production clusters. As such, the wording of the title is a little confusing to me at present. Would it make sense to rename this to be something similar to:

> Configuring storage for the image registry in non-production clusters

I think the `non-production storage` bit is what's bothering me.

Comment (huffmanca) [reply modules/installation-registry-storage-non-production.adoc]: Or possibly:
> Configuring an empty directory for non-production clusters

**Review by huffmanca — COMMENTED** [review]
One nitpick. I think it's fine if you want to merge it as is, though. The rest looks good to me!

File: modules/installation-registry-storage-non-production.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/installation-registry-storage-non-production.adoc]: I like the way the first one sounds. I'll update the file. Thank you!

**Comment by kalexand-rh** [comment]
@jianlinliu, will you PTAL? I moved some of the relevant options for configuring registry storage into the installation assemblies. I didn't modify the methods that were already approved. Do you agree with this change?

File: modules/registry-configuring-storage-baremetal.adoc [hunk 2]
```diff
@@ -24,7 +25,7 @@ the `configs.imageregistry/cluster` resource.
 . Verify you do not have a registry pod:
 +
 ----
-$ oc get pods -n configs.imageregistry.operator.openshift.io/cluster 
+$ oc get pods -n configs.imageregistry.operator.openshift.io/cluster
```
Comment (jianlinliu) [inline modules/registry-configuring-storage-baremetal.adoc]: Obviously the above command is using wrong syntax. 
Here you want to verify if a image registry config - 'cluster' is existing for the following modification, but not pod. am I right?
If you want to verify registry existing or not, should be:
`$ oc get po -n openshift-image-registry`

File: modules/registry-configuring-storage-vsphere.adoc [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 //* architecture/installation-.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
 
 [id="registry-configuring-storage-vsphere_{context}"]
 = Configuring registry storage for vSphere
```
Comment (jianlinliu) [inline modules/registry-configuring-storage-vsphere.adoc]: The same issue like baremetal image registry configuration.
Obviously the above command is using wrong syntax. 
`
$ oc get pod -n configs.imageregistry.operator.openshift.io/cluster
invalid namespace "configs.imageregistry.operator.openshift.io/cluster": [may not contain '/']
`
Here you want to verify if a image registry config - 'cluster' is existing for the following modification, but not pod. am I right?
If you want to verify registry existing or not, should be:
`$ oc get po -n openshift-image-registry`

File: modules/registry-configuring-storage-aws-user-infra.adoc [hunk 4]
```diff
@@ -21,7 +22,7 @@ procedure.
 Use the following procedure if the Registry Operator cannot create an S3 bucket
 and automatically configure storage.
 
-. Create a S3 bucket:
+. Create an S3 bucket:
 +
 ----
 $ oc create configs.imageregistry.operator.openshift.io
```
Comment (jianlinliu) [inline modules/registry-configuring-storage-aws-user-infra.adoc]: I do not think this command is used for creating an s3. s3 should be created manually by user in this case from aws cli or web console, isn't it?
And even this command can not be executed successfully.
`$ oc create configs.imageregistry.operator.openshift.io
Error: required flag(s) "filename" not set`

Comment (kalexand-rh) [reply modules/registry-configuring-storage-aws-user-infra.adoc]: Brandi says that it's fine to remove this command.

**Comment by kalexand-rh** [comment]
@jianlinliu, thanks! I updated the commands per your comments. Brandi says that the original commands were approved by QE earlier in the release, but if yours are right now, to use them.

**Comment by jianlinliu** [comment]
LGTM.

**Comment by kalexand-rh** [comment]
Thanks! Merging.

**Pull request merged**